common/src/chain/config/mod.rs

```diff
@@ -329,6 +329,16 @@ impl ChainConfig {
         }
     }
 
+    #[must_use]
+    pub fn order_id_address_prefix(&self) -> &'static str {
+        match self.chain_type {
+            ChainType::Mainnet => "mordr",
+            ChainType::Testnet => "tordr",
+            ChainType::Regtest => "rordr",
+            ChainType::Signet => "sordr",
+        }
+    }
+
```

Comment:
do we want a fork with activation for this functionality?

Reply:
Yes